packages/coil-extension/manifest.json

```diff
@@ -32,5 +32,5 @@
       "run_at": "document_start"
     }
   ],
-  "permissions": [ "webNavigation", "<all_urls>" ]
+  "permissions": [ "webNavigation", "<all_urls>", "tabs" ]
```

Comment:
Chrome (and to a lesser extent, Firefox) grants many "tabs" permissions implicitly via 
"<all_urls>". Safari does not. This is required for using chrome.tabs to search for an already open tab via url and select it.